mlflow/store/artifact/databricks_models_artifact_repo.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import json
+
+import mlflow.tracking
+from mlflow.entities import FileInfo
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+from mlflow.store.artifact.artifact_repo import ArtifactRepository
+from mlflow.utils.databricks_utils import get_databricks_host_creds
+from mlflow.utils.file_utils import download_file_using_http_uri
+from mlflow.utils.rest_utils import http_request
+from mlflow.utils.uri import get_databricks_profile_uri_from_artifact_uri
+from mlflow.store.artifact.utils.model_utils import (
+    get_model_name_and_version,
+    is_using_databricks_registry,
+)
+
+_logger = logging.getLogger(__name__)
+_DOWNLOAD_CHUNK_SIZE = 100000000
```

Comment:
How did you specify this number? Shouldn't it be 2 powered value at least?

Reply:
At the time the `DatabricksArtifactRepository` was added, we could upload to Azure in chucks of 100 MB. So we just used the same chunk size to download these artifacts.
Also we use response.iter_content to download a file as downloading large files could result in OOM errors. So 100 MB seemed decent to use. Can we changed if you feel strongly about this.